src/Element/Element.php

```diff
@@ -204,6 +196,6 @@ public function getOuterHtml()
      */
     protected function elementNotFound($type, $selector = null, $locator = null)
```

Comment:
Isn't this just an internal method that contained some logic before, but not now? If so, then we'd better show exception where needed instead of calling method, that throws them for us. This would also give us shorter stacktrace for that exception.